src/task/tests/pullRequests/pullRequest.spec.ts

```diff
@@ -42,14 +45,17 @@ describe('pullRequest.ts', (): void => {
     when(runnerInvoker.loc('pullRequests.pullRequest.titleFormat', 'S✔', 'PS✔ ◾ Title')).thenReturn('S✔ ◾ PS✔ ◾ Title')
     when(runnerInvoker.loc('pullRequests.pullRequest.titleFormat', 'S✔', 'PS⚠️ ◾ Title')).thenReturn('S✔ ◾ PS⚠️ ◾ Title')
     when(runnerInvoker.loc('pullRequests.pullRequest.titleFormat', '(XS|S|M|L|\\d*XL)(✔|⚠️)?', '(.*)')).thenReturn('(XS|S|M|L|\\d*XL)(✔|⚠️)? ◾ (.*)')
+
+    inputs = mock(Inputs)
+    when(inputs.changeTitle).thenReturn(InputsDefault.changeTitle)
```

Comment:
Please move this to the area around Line 24, to keep the ordering consistent. In my experience, keeping this consistent allows for quicker reasoning about the class by knowing where something should be based on where it is in other parts of the class.